.env_sample

```diff
@@ -0,0 +1,31 @@
+# TO USE THIS FILE RENAME IT TO '.env'
+
+# NOTE! Using this file requires `pip install python-dotenv`
```

Comment:
I love dotenvs, especially as practical ways of keeping an inventory of our environment variables